packages/cli/snapshots/integration/generators/relation.belongs-to.integration.snapshots.js

```diff
@@ -255,7 +259,7 @@ export class Order extends Entity {
   name?: string;
 
   @belongsTo(() => Customer)
-  myRelation: number;
+  customerId: number;
```

Comment:
Is this change intentional and legitimate? I have no idea, just checking to ensure we don't accidentally overlook a regression.

Reply:
This test aims to check if the pre-exist property `@property{} something;` will be overwritten by `@belongsTo` decorator. That's why I use the default relation name to make the intention more clear.